epsagon/utils.py

```diff
@@ -109,6 +110,8 @@ def init(
       collection.
     :param keys_to_ignore: List of keys to ignore while extracting metadata.
     :param ignored_endpoints: List of ignored endpoints for web frameworks.
+    :param split_on_send: Split the trace on send flag
+    :param propagate_id: Inject identifiers via return value flag
```

Comment:
It's relevant only for Lambda, and I don't see that it gonna change, maybe update the name? (Also the env var name?)